ghostjs-examples/test/network_issue_test.js

```diff
@@ -0,0 +1,26 @@
+import ghost from 'ghostjs'
+import assert from 'assert'
+
+import localServer from './fixtures/server.js'
+
+describe('Returns after a network issue', () => {
+  before(localServer)
+  after(localServer.stop)
+
+  it('hits a connection refused and doesnt time out', async () => {
+    let flag;
+    const raceServerResponse = setTimeout(() => {
+      // If this setTimeout beat the error response, we're waiting too long
+      flag = false
+    }, 90000)
```

Comment:
Is this still deterministic if we make this 30s or so? Ghost CI doesn't have great parallelism because we just use travis, so I am still a bit conscious on run time. Just wondering if we can use a shorter time here.